.github/workflows/test.yml

```diff
@@ -76,4 +76,4 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: lcov.info
-          fail_ci_if_error: false
\ No newline at end of file
+          fail_ci_if_error: false
```

Comment:
does this setting mean that if tests fail, the CI won't fail? i don't think so, if not, what it is for?

Reply:
I think that's just for the test coverage, so yes indeed it doesn't fail on test coverage.